pages/index.js

```diff
@@ -1,7 +1,40 @@
+import Image from 'next/image';
+
 export default function Home() {
+  console.log('hello');
```

Comment:
In order to follow JavaScript best practices, you should avoid pushing `console.log` in the repo